permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -147,45 +152,45 @@ class Permission {
   /// https://support.google.com/googleplay/android-developer/answer/9214102#zippy=
   static const manageExternalStorage = Permission._(22);
 
-  ///Android: Allows an app to create windows shown on top of all other apps
-  ///iOS: Nothing
+  /// Android: Allows an app to create windows shown on top of all other apps
+  /// iOS: Nothing
   static const systemAlertWindow = Permission._(23);
 
-  ///Android: Allows an app to request installing packages.
-  ///iOS: Nothing
+  /// Android: Allows an app to request installing packages.
+  /// iOS: Nothing
   static const requestInstallPackages = Permission._(24);
 
-  ///Android: Nothing
-  ///iOS: Allows user to accept that your app collects data about end users and
-  ///shares it with other companies for purposes of tracking across apps and
-  ///websites.
+  /// Android: Nothing
+  /// iOS: Allows user to accept that your app collects data about end users and
+  ///   shares it with other companies for purposes of tracking across apps and
+  ///   websites.
   static const appTrackingTransparency = Permission._(25);
 
-  ///Android: Nothing
-  ///iOS: Notifications that override your ringer
+  /// Android: Nothing
+  /// iOS: Notifications that override your ringer
   static const criticalAlerts = Permission._(26);
 
-  ///Android: Allows the user to access the notification policy of the phone.
+  /// Android: Allows the user to access the notification policy of the phone.
   /// EX: Allows app to turn on and off do-not-disturb.
-  ///iOS: Nothing
+  /// iOS: Nothing
   static const accessNotificationPolicy = Permission._(27);
 
-  ///Android: Allows the user to look for Bluetooth devices
-  ///(e.g. BLE peripherals).
-  ///iOS: Nothing
+  /// Android: Allows the user to look for Bluetooth devices
+  ///   (e.g. BLE peripherals).
+  /// iOS: Nothing
   static const bluetoothScan = Permission._(28);
 
-  ///Android: Allows the user to make this device discoverable to other
-  ///Bluetooth devices.
-  ///iOS: Nothing
+  /// Android: Allows the user to make this device discoverable to other
+  ///   Bluetooth devices.
```

Comment:
nit: redundant spacing:
```suggestion
  /// Bluetooth devices.
```